server/config.go

```diff
@@ -67,6 +67,7 @@ func ParseConfig(bytes []byte) (*Config, error) {
 	}
 
 	c.Options.FillDefaults()
+	c.Server.SetValuesFromEnv("")
```

Comment:
As I mentioned on the `go-baseapp` PR, I think we should use `POLICYBOT_` as the prefix for these values. The GitHub values below can stay as-is, since they already use `GITHUB_` as a prefix.